src/dom-to-image-more.js

```diff
@@ -74,22 +75,46 @@
         const ownerWindow = domtoimage.impl.util.getWindow(node);
         options = options || {};
         copyOptions(options);
+        let restorations = [];
         return Promise.resolve(node)
+            .then(ensureElement)
             .then(function (clonee) {
                 return cloneNode(clonee, options, null, ownerWindow);
             })
             .then(embedFonts)
             .then(inlineImages)
             .then(applyOptions)
-            .then(function (clone) {
```

Comment:
This was kind of ugly before, not the normal `then()` pattern.  ALSO fixed the clipping bug where, because previously we were sizing things by the original `node`'s dimensions, when we should have been using the `clone`'s dimensions.